api/v1alpha2/eip_types.go

```diff
@@ -127,19 +128,24 @@ func (e Eip) ValidateCreate() error {
 	if err != nil {
 		return err
 	}
-
+	defaultEipCount := false
```

Comment:
change `defaultEipCount` to `existDefaultEip`

Reply:
Agree to lxm. I have no problem with the other parts

Reply:
ok